pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,310 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	kapierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kbatch "k8s.io/kubernetes/pkg/apis/batch"
+	kclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/host"
+)
+
+const (
+	defaultAnsibleImageUser        = "root"
+	defaultOpenshiftAnsibleImage   = "ansible"
+	deploymentTypeOrigin           = "origin"
+	deploymentTypeOCP              = "openshift-enterprise"
+	imageStreamCentos              = "centos7"
+	openshiftAnsibleServiceAccount = "openshift-ansible"
+)
+
+const defaultMetricsInventory = `
+[OSEv3:children]
+masters
+nodes
+
+[OSEv3:vars]
+#openshift_release={{.OSERelease}}
+
+openshift_deployment_type={{.OSEDeploymentType}} 
+openshift={master:{ha: false}}
+
+openshift_metrics_install_metrics=True
+openshift_metrics_image_prefix={{.MetricsImagePrefix}}
+openshift_metrics_image_version={{.MetricsImageVersion}}
+openshift_metrics_resolution={{.MetricsResolution}}
+
+openshift_metrics_hawkular_hostname={{.HawkularHostName}}
+
+[masters]
+{{.MasterIP}} ansible_connection=local
+
+[nodes]
+{{.MasterIP}}
+`
+
+const defaultLoggingInventory = `
+[OSEv3:children]
+masters
+nodes
+
+[OSEv3:vars]
+#openshift_release={{.OSERelease}}
+
+openshift_deployment_type={{.OSEDeploymentType}} 
+openshift={master:{ha: false}}
+
+openshift_hosted_logging_deployer_prefix={{.LoggingImagePrefix}}
+openshift_hosted_logging_deployer_version={{.LoggingImageVersion}}
+openshift_hosted_logging_master_public_url={{.MasterPublicURL}}
+
+openshift_logging_install_logging=true
+openshift_logging_use_ops=false
+openshift_logging_namespace={{.LoggingNamespace}}
+
+openshift_logging_elasticseach_memory_limit=1024M
+openshift_logging_elasticseach_storage_type=pvc
+openshift_logging_elasticseach_pvc_size=100G
+
+openshift_logging_kibana_hostname={{.KibanaHostName}}
```

Comment:
We will eventually deprecate the openshift_hosted_logging variables, so lets use the equivalents from https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/README.md rather than the old ones.